File: app/src/main/java/lab/inconcept/musiclibrary/helper/PreferenceManager.java
package lab.inconcept.musiclibrary.helper;

import android.content.Context;
import android.content.SharedPreferences;

public final class PreferenceManager {
    private static final String PREFERENCES_TAG = "preferences";

    public static void putString(Context context, String key, String value) {
        final SharedPreferences sPref = context.getSharedPreferences(PREFERENCES_TAG, Context.MODE_PRIVATE);
        final SharedPreferences.Editor ed = sPref.edit();
        ed.putString(key, value);
        ed.apply();
    }

    public static String getString(Context context, String key, String defValue) {
        final SharedPreferences sPref = context.getSharedPreferences(PREFERENCES_TAG, Context.MODE_PRIVATE);
        return sPref.getString(key, defValue);
    }

}

File: app/src/main/java/lab/inconcept/musiclibrary/helper/PreferenceManager.kt
package lab.inconcept.musiclibrary.helper

import android.content.Context

object PreferenceManager {
    private const val PREFERENCES_TAG = "preferences"

    fun putString(context: Context, key: String?, value: String?) {
        val sPref = context.getSharedPreferences(PREFERENCES_TAG, Context.MODE_PRIVATE)
        val ed = sPref.edit()
        ed.putString(key, value)
        ed.apply()
    }

    fun getString(context: Context, key: String?, defValue: String?): String? {
        val sPref = context.getSharedPreferences(PREFERENCES_TAG, Context.MODE_PRIVATE)
        return sPref.getString(key, defValue)
    }
}